### PR TITLE
#9Task. Minko Stas

### PR DESCRIPTION
**Pull request by StanislavM31**
https://travis-ci.com/github/StanislavM31/js-assignments/builds/211273943

### CODE REVIEW

**Comment by StanislavM31** [comment]
@vramaniuk 
done as required, can you check, pls?

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -26,7 +26,7 @@
  *
  */
 function getComposition(f,g) {
-    throw new Error('Not implemented');
+    return (x) => f(g(x));
```
Comment (vramaniuk) [inline task/09-functions-n-closures-tasks.js]: Please, spaces around nested call
https://javascript.info/coding-style
Selection_010

File: task/09-functions-n-closures-tasks.js [hunk 2]
```diff
@@ -65,7 +65,14 @@ function getPowerFunction(exponent) {
  *   getPolynom()      => null
  */
 function getPolynom() {
-    throw new Error('Not implemented');
+    if (arguments.length === 0) return null;
+    return (x) => {
+        let y = 0;
+        for (let i = 0; i < arguments.length; i++) {
+            y += arguments[i]*Math.pow(x, arguments.length - i - 1);
```
Comment (vramaniuk) [inline task/09-functions-n-closures-tasks.js]: Please, spaces around operators
https://javascript.info/coding-style

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Comment by StanislavM31** [comment]
@vramaniuk, and what is now? Can you connect me to the third level in the slack?? The necessary level to go to the next level is: 9 completed tasks and laid out messenger UI (I made a telegram). I was filling out a google form to validate my UI some weeks ago.

**Comment by StanislavM31** [comment]
@vramaniuk, @IlyaMokin, I just filled out the google form again (as it was written in the description of the level 2 task)

**Pull request closed without merging**